sjsonnet/test/src-jvm/sjsonnet/ErrorTests.scala

```diff
@@ -303,5 +303,8 @@ object ErrorTests extends TestSuite{
 //        |    at .(sjsonnet/test/resources/test_suite/error.invariant.simple3.jsonnet:18:10)
 //        |""".stripMargin
 //    )
+    test("duplicate_fields") - check(
```

Comment:
Could we put this in `EvaluatorTests` instead? Currently all the test cases in `test_suite/` that are used in `FileTests.scala` and `ErrorTests.scala` are directly vendored from the google/jsonnet implementation, so it's best to keep our own custom additions separate so next time we update the vendored code we don't accidentally lose our additions

Reply:
`EvaluatorTests` seems like a strange place for a parser test.

I also assumed that the test suite should be the original Jsonnet one, but then I saw that it had a few changes applied over time and it doesn't match the current Jsonnet test suite.

Reply:
How about `ParserTests`? Not much in there now, but if this is a parser test then it seems like it belongs